examples/databricks-secret.yaml

```diff
@@ -0,0 +1,27 @@
+---
+apiVersion: com.dstancu.databricks/v1
+kind: DatabricksSecretScope
+metadata:
+  name: my-secret-scope
+  namespace: default
+spec:
+  scope:
+    name: my-super-cool-scope
+---
+apiVersion: v1
+data:
+  hello: d3VybA==
+  foo: YmFy
+kind: Secret    
+metadata:
+  name: my-super-cool-secret
+type: Opaque
+---
+apiVersion: com.dstancu.databricks/v1
+kind: DatabricksSecret
+metadata:
+  name: my-super-cool-databricks-secret
+  namespace: default
+spec:
```

Comment:
A secret is the combination of a scope and Kubernetes secret. An instance of `DatabricksSecret` means "mirror this Kubernetes secret into the Databricks API".